agenta-web/src/components/Playground/Views/TestView.tsx

```diff
@@ -174,10 +181,21 @@ const App: React.FC<TestViewProps> = ({inputParams, optParams, URIPath}) => {
 
     const handleAddRow = () => {
         setTestList([...testList, {}])
+        setResultsList([...testList, {}].map(() => ""))
     }
 
     const handleSetNewTests: (tests: Record<string, string>[]) => void = (tests) => {
         setTestList([...tests])
+        setResultsList(tests.map(() => ""))
+    }
+
+    const handleDeleteRow = (testIndex: number) => {
+        if (resultsList.length < 2) return
+        if (resultsList[testIndex] != "") {
```

Comment:
Please always perform type sensitive checks (!== and ===), unless there is a good reason not to